src/main/kotlin/no/nav/amt/distribusjon/journalforing/model/Endringshendelse.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.amt.distribusjon.journalforing.model
+
+import no.nav.amt.distribusjon.hendelse.model.Hendelse
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class Endringshendelse(
+    val hendelseId: UUID,
+    val deltakerId: UUID,
+    val hendelse: Hendelse,
```

Comment:
Enda en grunn til at jeg sliter litt med å forstå denne, er at man har allerede `hendelseId` `deltakerId` og `opprettet` i `Hendelse` så det virker litt overflødig med å ha det dobbelt opp.

Reply:
Kunne vi bare ha lagret hendelsen i en spesifikk endrings-tabell, også greid oss uten denne?

Reply:
f.eks:
```create table endringshendelse
(
    id uuid primary key,
    deltaker_id uuid                                               not null, --for index sin del er det greit med dobbelt opp
    deltaker jsonb not null,
    ansvarlig jsonb not null,
    payload    jsonb                                              not null,
    distribusjonskanal varchar,
    created_at  timestamp with time zone default current_timestamp not null
);```

Jeg tror det kan være behov for å kunne legge til felter som `distribusjonskanal` og annet rundt hendelsene, så det kan være greit å ikke ha alt som en stor json.